nightfall-client/src/routes/commitment.mjs

```diff
@@ -88,6 +90,41 @@ router.get('/commitments', async (req, res, next) => {
   }
 });
 
+/**
+ * @description the endpoint that will send a reponse with all the
+ * existent commitments for the list of compressedPkd received in the
+ * request body.
+ * @author luizoamorim
+ */
+router.post('/byCompressedZkpPublicKey', async (req, res, next) => {
```

Comment:
```suggestion
router.post('/compressedZkpPublicKeys', async (req, res, next) => {
```

Please update the route :pray:

Reply:
Like Israel suggested, please add a comment explaining why this is POST instead of GET